src/sentry/options/defaults.py

```diff
@@ -296,20 +296,6 @@
     default=False,
     flags=FLAG_MODIFIABLE_BOOL | FLAG_AUTOMATOR_MODIFIABLE,
 )
-# Enable EA endpoints to work with id or slug as path parameters
-register(
```

Comment:
Do you need to remove the options from the automator first?

Reply:
i have that pr ready to be merged, i just wanted to make this pr to figure out if any tests fail because i missed something.

Reply:
https://github.com/getsentry/sentry-options-automator/pull/1493

Reply:
I merged the above PR, will merge this tomorrow if we are good.